tests/test_ranges.py

```diff
@@ -1243,19 +1252,6 @@ def test_returns_date_range(self):
                 datetime.date(2020, 1, 9),
             )
 
-        def test_errors_if_different_timezones(self):
-            dt_range = ranges.FiniteDatetimeRange(
-                datetime.datetime(2020, 1, 1, tzinfo=zoneinfo.ZoneInfo("Asia/Dubai")),
-                datetime.datetime(
-                    2020, 1, 10, tzinfo=zoneinfo.ZoneInfo("Australia/Sydney")
-                ),
-            )
```

Comment:
Removing this test case since it's no longer possible to construct such a range!